homeassistant/components/light/hue.py

```diff
@@ -27,6 +29,10 @@
 
 PHUE_CONFIG_FILE = "phue.conf"
 
+SUPPORT_HUE = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT |
```

Comment:
I am not entirely sure what the capabilities are intended to do, but not all lamps withing the Hue eco-system support all these attributes. For example the Hue White has only brightness, effect and transition. Also these capabilities seem to be already exposed somehow to the frontend since the controls I get in the UI are different for the lamps capabilities.